app/assets/javascripts/kassi.js

```diff
@@ -680,31 +685,39 @@ function initialize_listing_view(locale) {
   });
 }
 
-function initialize_accept_transaction_form(commission_percentage, service_fee_vat, form_type, form_id, minimum_price, minimum_price_message) {
-	auto_resize_text_areas("text_area");
-	style_action_selectors();
-
-	if (commission_percentage != null) {
-	  if (form_type === "simple") {
-	    $(".trigger-focusout").keyup(function(value) {
-	      update_simple_form_price_fields(commission_percentage);
-	    });
-	    $(form_id).validate({
-	      rules: {
-          "listing_conversation[payment_attributes][sum]": {money: true, minimum_price_required: minimum_price}
+function initialize_accept_transaction_form(
+  commission_percentage,
+  service_fee_vat,
+  form_type,
+  form_id,
+  minimum_price,
+  subunit_to_unit,
+  minimum_price_message) {
+
+  auto_resize_text_areas("text_area");
+  style_action_selectors();
+
+  if (commission_percentage != null) {
```

Comment:
Maybe, but modification to this function is just collateral damage from adding subunit parsing to listing create/edit so I don't feel like putting effort into refactoring this. Also, it's part of almost-deprecated postpay flow.